src/scenes/home/benefit/benefit.css

```diff
@@ -36,6 +36,17 @@
     width: 300px;
   }
 }
+@media screen and (max-width: 400px) {
+    div span:last-of-type {
+        display: block;
+    }
+}
+
+@media screen and (max-width: 350px) {
+    div span:nth-of-type(2) {
+        display: block;
+    }
+}
```

Comment:
I'm wondering what is the motivation behind specifying the spans instead of selecting any span and change the display?

Reply:
I don't understand what you are asking? 
The text doesn't wrap on small screens (specifically, iPhone5), so I was trying to get the text to break in a logical place. Is there a better way to do this?

Reply:
Sorry for not being clear.
Overall, I really liked the idea of separating the text into 2 spans.
I just wonder why didn't you just use `span` in the css file instead of `span:nth-of-type(2)` and `span:last-of-type`

Reply:
@ksmacleod99 We shouldn't be targeting HTML tags like `span` directly with CSS rules.  The above CSS could allow those style rules to apply to spans in other components in the app (not just benefit.css), causing unwanted side-effects.

I suggest giving adding appropriately-named classes to the spans that you want to style inside of the JSX in benefit.js, using the `className` attribute.  Then, you can adding the specific style rules for those classes in benefit.css.  CSS Modules will ensure that those rules will be scoped only to elements w/ the matching `className` inside the Benefit component.  

We want to avoid repeats of issues #511 and #555.

Reply:
Tracking. I will work on those tomorrow. Thank you for the review.